File: SWEA/D1/2056. 연월일 달력/연월일 달력.cpp
T = int(input())
for t in range(1, T+1):
    n = input()
    year = n[0:4]
    month = n[4:6]
    day = n[6:8]

    if month == '00' or month > '12':
        print(f'#{t} -1')
    else:
        # print('1')
        if month == '02':
            #print('1')
            if day > '28':
                print(f'#{t} -1')
            else:
                print(f'#{t} {year}/{month}/{day}')
        else:
            print(f'#{t} {year}/{month}/{day}')

File: SWEA/D1/2056. 연월일 달력/연월일 달력.py
import datetime
T = int(input())
for t in range(1, T+1):
    num = input()
    year, month, day = num[:4], num[4:6], num[6:]
    try:
        datetime.date(int(year), int(month), int(day))
        print(f"#{t} {year}/{month}/{day}")
    except:
        print(f"#{t} -1")